bee-ledger/src/workers/consensus/white_flag.rs

```diff
@@ -77,9 +77,6 @@ fn apply_regular_essence<B: StorageBackend>(
         let _essence_hash = TransactionEssence::from(essence.clone()).hash();
 
         match consumed_output.inner() {
-            Output::Simple(_) => {
-                // TODO
-            }
```

Comment:
Nice, one thing less to implement :trollface:

Reply:
one DTO less in the API :trollface: